### PR TITLE
Contact Form 7 integration (WIP)

### PR DESCRIPTION
**Pull request by ramiabraham**
- Inserts pending referral
- Completes or rejects referral depending on success/cancel PayPal return url.
- ~~Requires Flamingo add-on~~
- #1336

### CODE REVIEW

File: includes/integrations/class-contactform7.php [hunk 1]
```diff
@@ -49,81 +58,241 @@ public function init() {
...
+	/**
+	 * Register the Contact Form 7 integration settings tab
+	 *
+	 * @access public
+	 * @since  2.0
+	 * @return array The new tab name
+	 */
+	public function register_settings_tab( $tabs = array() ) {
+
+		$tabs['contactform7'] = __( 'Contact Form 7', 'affiliatewp-zapier' );
```
Comment (pippinsplugins) [inline includes/integrations/class-contactform7.php]: Incorrect text domain here.

File: includes/integrations/class-contactform7.php [hunk 2]
```diff
@@ -49,81 +58,241 @@ public function init() {
...
+					 * @since  1.0
+					 */
+					esc_url( apply_filters( 'afwp_cf7_admin_docs_url', $doc_url ) ),
+					'<span class="dashicons dashicons-external"></span>',
+					'<span class="dashicons dashicons-info"></span>'
+				),
+				'type' => 'checkbox'
+			),
+			'affwp_cf7_enable_specific_forms' => array(
+				'name' => '<strong>' . __( 'Enable referrals for specific Contact Form 7 forms', 'affiliatewp-zapier' ) . '</strong>',
```
Comment (pippinsplugins) [inline includes/integrations/class-contactform7.php]: Incorrect text domain

File: includes/integrations/class-contactform7.php [hunk 3]
```diff
@@ -0,0 +1,913 @@
...
+				$all_forms[ $post_id ] = get_the_title();
+			}
+		}
+
+		wp_reset_postdata();
+
+		return $all_forms;
+	}
+
+	public function all_forms_multicheck_render() {
```
Comment (pippinsplugins) [inline includes/integrations/class-contactform7.php]: Let's add PHPdoc here please.

File: includes/integrations/class-contactform7.php [hunk 4]
```diff
@@ -0,0 +1,913 @@
...
+	 * @see    affwp_cf7_ajax
+	 */
+	public function ajax_get_paypal_meta(){
+
+		if ( isset( $_REQUEST ) ) {
+			$form_id = $_REQUEST['form_id'];
+
+		}
+
+	    $enabled     = get_post_meta( $form_id, '_cf7pp_enable', true );
```
Comment (pippinsplugins) [inline includes/integrations/class-contactform7.php]: Looks like our indention is incorrect here and below on line 593

Comment (ramiabraham) [inline includes/integrations/class-contactform7.php]: Corrected as noted

File: includes/integrations/class-contactform7.php (hunk 3 above)
Comment (ramiabraham) [inline includes/integrations/class-contactform7.php]: Added as noted

**Pull request merged**